app/views/worklogs/_below_worklog_index.html.slim

```diff
@@ -13,7 +13,7 @@
         p.lead.pull-right Client
       .col-md-10
         select.col-md-4.client-filter.form-control
-          option data-url="#{worklogs_path(merge_params(client: nil))}"
+          option data-url="#{worklogs_path(merge_params(client: nil))}" value="" ng-if="false"
```

Comment:
I tested it locally and it worked fine. The option comes from stackoverflow and got approved by others as working as well :)